doc/requirements/io_class

```
@@ -0,0 +1,86 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy_per_ioclass
+---
+
+It shall be possible to set a maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Occupancy limit interpretation
+id: occupancy_per_ioclass_limit_interpretation
+---
+
+The product shall shall interpret max occupancy value as a percentage of
+a cache size
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: occupancy_per_ioclass_allowed_values
+---
+
+The product shall accept values for a max size of an IO class in range from 0 to 100
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default occupancy limit
+id: occupancy_per_ioclass_default_limit
+---
+
+When initializing a cache the software shall set default IO class's max occupancy
```

actually the word "default" is misleading here, I suggest to remove it